scripts/build/docker-build-image

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+VOLUMES="-v /sys:/sys -v /home:/mnt/host/home"
+IMAGE=$1
+DOCKERFILE=$IMAGE
+shift
+
+while [ -n "$1" ]; do
+    case $1 in
+        --volume|-v)
+            VOLUMES="$VOLUMES -v $2"
+            shift 2
+            ;;
+        --container|-c)
+            if [ -n "$2" -a "${2#-}" = "$2" ]; then
+                CONTAINER="$2"
+                shift
+            else
+                CONTAINER="$IMAGE"
+            fi
+            shift
+            ;;
+        --dockerfile|-df)
+            DOCKERFILE=$2
+            shift 2
+            ;;
+        *)
+            PASSTHROUGH="$PASSTHROUGH $1"
+            shift
+            ;;
+    esac
+done
```

Comment:
I'd rather leave it like this for consistency since all of our other scripts parse the command line in a similar manner. It'd need to be a separate PR converting all of those to getopts (but to be honest I doubt if it's worth the effort).

Reply:
Fair enough, that's a helper script, anyway